impeller/golden_tests/main.cc

```diff
@@ -24,7 +25,14 @@ void print_usage() {
 }
 }  // namespace
 
+namespace impeller {
+TEST(ValidationTest, IsFatal) {
+  EXPECT_TRUE(ImpellerValidationErrorsAreFatal());
```

Comment:
I tried to use `EXPECT_EXIT` but I couldn't get it to stop killing the process.

Reply:
You want to use EXPECT_DEATH? 

That seems to work for me